rpc/src/services/protocol/proto_006/mod.rs

```diff
@@ -1,6 +1,81 @@
 // Copyright (c) SimpleStaking, Viable Systems and Tezedge Contributors
 // SPDX-License-Identifier: MIT
 
+use getset::CopyGetters;
+use num::BigInt;
+use serde::Deserialize;
+
 mod helpers;
 pub(crate) mod rights_service;
 pub(crate) mod votes_services;
+
+use super::{string_to_int, vec_string_to_int};
+
+#[derive(Debug, Deserialize, Clone, CopyGetters)]
+pub(crate) struct ProtocolConstants {
```

Comment:
@adonagy 
are these `ProtocolConstants` structs different from those constants from `tezos_messages/src/protocol/proto_*`,
couldnt be better reused? is it possible?

Reply:
Same reason as here: https://github.com/tezedge/tezedge/pull/782#discussion_r710857033